### PR TITLE
Sort list of commands. Fixes #74.

### LINKED ISSUE #74: Display all tasks list in alphabetically sorted manner
Currently when displaying all tasks list by `npx run`, tasks list is unsorted. It should be sorted alphabetically.

### PR DESCRIPTION
**Pull request by markstos**
Manually tested before/after.

Sorry about all the whitespace changes included-- the "prettier" integration insisted I make the changes.

All I did was add a well-placed `.sort()` call. The rest is formatting.

### CODE REVIEW

**Comment by markstos** [comment]
I've now pushed an update here which includes a test.

I also have a co-worker, @GrantHeger who is the process of moving to Melbourne. Perhaps you'll run into him some day!

**Comment by pawelgalazka** [comment]
@markstos I merged one of your PR, but it seems that caused conflict here. Can you quickly resolve it ? Then I will try to merge this one

**Comment by markstos** [comment]
Resolved.

File: src/index.js [hunk 1]
```diff
@@ -104,6 +107,9 @@ export function run(
   }
 
   // Handle sync call by default
+  if (typeof options.input !== 'undefined') {
+    nextOptions.input = options.input
+  }
```
Comment (pawelgalazka) [inline src/index.js]: This change doesn't seems to be as part of this PR. Same in README. Let's keep PR atomic to the change which is the main subject

**Comment by pawelgalazka** [comment]
This has been already resolved

**Pull request closed without merging**